app/src/pages/profile/ProfilePage.tsx

```diff
@@ -45,8 +109,38 @@ export function ProfilePage() {
               />
             </Form>
           </Card>
-          <Card title="API Keys">user api key</Card>
+          <Card
+            title="API Keys"
+            variant="compact"
+            bodyStyle={{ padding: 0 }}
+            extra={
+              <Button
+                variant="default"
+                size="compact"
+                icon={<Icon svg={<Icons.PlusCircleOutline />} />}
+                onClick={() =>
+                  setDialog(
+                    <CreateAPIKeyDialog
+                      onSubmit={onSubmit}
+                      isCommitting={isCommitting}
+                    />
+                  )
+                }
+              >
+                New Key
+              </Button>
+            }
+          >
```

Comment:
Let's move this card into a component so the mutations are encapsulated - e.g. ViewerApiKeys